### ISSUE TITLE
[fatal] WARNNING!!! Licode not work! (chrome between before  and after M74 )

### ISSUE THREAD

**Issue by lidedongsn** [lynckia/licode#1399]
try comment file rtp_media_config.js
```
//"urn:3gpp:video-orientation",
```
publish stream in chrome 版本 73.0.3683.103（正式版本） （64 位）
subscribe in chrome 版本 75.0.3766.2（正式版本）canary （64 位）
**no subscribed stream!!!**

I'll explain why comment this for a test later.
I know that something about ExtensionId, I'll try my test, and give a PR soon.

**Comment by lidedongsn** [comment]
版本 73.0.3683.103（正式版本） （64 位）answer sdp video extmap：
```
a=extmap:2 urn:ietf:params:rtp-hdrext:toffset
a=extmap:3 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:4 urn:3gpp:video-orientation
a=extmap:6 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
```
版本 75.0.3766.2（正式版本）canary （64 位） answer sdp video extmap:
```
a=extmap:3 urn:3gpp:video-orientation
a=extmap:5 http://www.webrtc.org/experiments/rtp-hdrext/playout-delay
a=extmap:13 http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
a=extmap:14 urn:ietf:params:rtp-hdrext:toffset
```
73 publish 75 subscribe,
captrue rtp using wireshark, a vp8 packet as example
before pass licode:
```
Header extensions
    RFC 5285 Header Extension (One-Byte Header)
        Identifier: 3
        Length: 3
        Extension Data: 0ac6a8

```
Identifier: 3 as http://www.webrtc.org/experiments/rtp-hdrext/abs-send-time
after pass licode to 75:
```
Header extensions
    RFC 5285 Header Extension (One-Byte Header)
        Identifier: 3
        Length: 3
        Extension Data: 0ac6a8

```
75 receive Identifier: 3 as  urn:3gpp:video-orientation

when comment comment file rtp_media_config.js
```
//"urn:3gpp:video-orientation",
```
75 will not receive Identifier: 3  rtp

It's wrong anyhow!
so need translate ExtensionId as needed
see PR #1400

**Comment by kekkokk** [comment]
this is a topic that was already discussed https://github.com/lynckia/licode/pull/1287 but was never merged.
Extension translation is crucial for interoperability with firefox and other browser since the ABS_SEND_TIME it's not currently translated resulting in a low bandwith estimation
Thanks for this warning @lidedongsn

**Comment by lidedongsn** [comment]
impact the use directly. No strategy?

**Comment by jcague** [comment]
PR in #1287 has same changes requested and they were never addressed. We are currently fixing negotiation issues and there are other things to do first too. But it's under our radar for sure. In the mean time any update on the PR will be welcome! thanks a lot!

**Comment by mkhahani** [comment]
It's 6 days that Chrome 74 has been released officially and Licode is almost useless with this issue. Every thing is messed up. Planning to migrate to another project 👎

**Comment by lidedongsn** [comment]
try PR #1400

**Comment by lidedongsn** [comment]
> It's 6 days that Chrome 74 has been released officially and Licode is almost useless with this issue. Every thing is messed up. Planning to migrate to another project

try PR #1400

**Comment by lodoyun** [comment]
Just merged #1400   this is fixed in my tests - thanks @lidedongsn
